src/sentry/south_migrations/0514_auto__del_grouptagvalue__del_unique_grouptagvalue_group_id_key_value__.py

```diff
@@ -0,0 +1,1565 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        # Deleting model 'GroupTagValue'
+        db.delete_table('sentry_messagefiltervalue')
+
+        # Deleting model 'TagKey'
+        db.delete_table('sentry_filterkey')
+
+        # Deleting model 'GroupTagKey'
+        db.delete_table(u'sentry_grouptagkey')
```

Comment:
mind using unicode everywhere ?

Reply:
Weird, I wonder why it got generated like this